app/views/devise/sessions/new.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <h1><%= t '.header' %></h1>
 <p><%= t '.description_html' %></p>
-<a href="/auth/CCID" class="btn btn-primary">
+<a href="/auth/saml" class="btn btn-primary">
```

Comment:
`omniauth-saml` doesn't support the custom naming implementation of `ominauth` which sucks. So reverted the custom name of CCID back to saml